rtos/TARGET_CORTEX/SysTimer.h

```diff
@@ -22,7 +22,7 @@
 #ifndef MBED_SYS_TIMER_H
 #define MBED_SYS_TIMER_H
 
-#if DEVICE_LPTICKER || defined(DOXYGEN_ONLY)
+#if MBED_TICKLESS || defined(DOXYGEN_ONLY)
```

Comment:
This impacts all the targets!
This should be checked very carefully, and should be out of this "L073RZ" PR...

Reply:
+1, PR introduction talks about fixing one target but there are fixes included impacting larger set of targets. Can we split these (l073 will be quick one to get in)

Reply:
Currently SysTimer cannot be built when lp ticker is disabled, leading to build errors of L073RZ. I'll move this SysTimer change to a separate pull request, and make the L073RZ change depend on it.